spark/src/main/scala/com/gu/tableversions/spark/VersionContext.scala

```diff
@@ -37,7 +37,8 @@ final case class SparkSupport(versionContext: VersionContext) {
         *         to the metastore.
         */
       def versionedInsertInto(table: TableDefinition, userId: UserId, message: String): (TableVersion, TableChanges) =
-        SparkSupport.versionedInsertDatasetIntoTable(versionContext, delegate, table, userId, message).unsafeRunSync()
+        F.toIO(SparkSupport.versionedInsertDatasetIntoTable(versionContext, delegate, table, userId, message))
```

Comment:
With `import cats.effect.implicits._` you'd be able to `.toIO.unsafeRunSync`, if you preferred (I'm fine with it as it is)

Reply:
Good to know!